index.html

```diff
@@ -11,42 +11,173 @@
+        <header class="nav">
+            <a class="logo-name" href="">DANIEL OCHUBA</a>
+                <div class="toggle-1"> 
+                    <div class="toggle"></div>
+                    <div class="toggle"></div>
+                    <div class="toggle"></div>
+                </div>
+        </header>
+        <main class="body">
+                        
+                        <div>
+                            <div class="section-2">
+                                <h1 class="primary-text">Hey There. I’m Daniel</h1>
+                                <h5 class="supporting-text">I’am a Software Developer</h5>
+                                <p class="description">I can help you build a product ,feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                                <div class="images">
+                                    <ul class="icons">  
+                                        <li class="icon"><a href=""><img src="./Disabled(2).png" alt=""></a></li>
+                                        <li class="icon"><a href=""><img src="./Disabled(1).png" alt=""></a></li>
+                                        <li class="icon"><a href=""><img src="./Enabled.png" alt=""></a></li>
+                                        <li class="icon"><a href=""><img src="./Disabled(3).png" alt=""></a></li>
+                                        <li class="icon"><a href=""><img src="./Disabled.png" alt=""></a></li>                        
+                                    </ul>
+                                </div> 
+                            
+                            </div>
+                            
+                        </div>
+                        
+        </main>
+        <div class="works-section">
+            <h1 class="title">My Recent Works</h1>
+            <hr class="rule">
+        </div> 
+        <div class="cards">
+            <div class="card-1">
+                <img src="./img_placeholder.png" alt="">
+                <h2 class="Title-post-1">Multi-Post Stories</h2>
+                <p class="Supporting-text-1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <div class="tags">
+                        <div class="tag-link-1">
+                            <span class="tag tag-link-1"><a href="https://www.w3.org/Style/CSS/Overview.en.html">css</a></span>
+                            <span class="tag tag-link-1"><a href="https://www.html.org">html</a></span>
+                            <span class="tag tag-link-1"><a href="https://getbootstrap.com/">bootstrap</a></span>
+                            <span class="tag tag-link-1"><a href="https://www.ruby-lang.org/en/">Ruby</a></span>
+                        </div>
```

Comment:
Well done on making some changes 💯  change is good. But there's more to be done, as mentioned by the previous reviewer the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") must be a list `ul`. and each of them a `li`. What for? 🤔 Semantics reasons and readability, and it also makes your life easier when it comes to styling them. 🕴🏽